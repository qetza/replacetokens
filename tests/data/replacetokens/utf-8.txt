utf-8
â‚¬
#{var1}#