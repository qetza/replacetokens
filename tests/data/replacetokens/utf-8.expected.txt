utf-8
â‚¬
var1_value